src/mumble/CoreAudio.h

```diff
@@ -6,50 +6,63 @@
 #ifndef MUMBLE_MUMBLE_COREAUDIO_H_
 #	define MUMBLE_MUMBLE_COREAUDIO_H_
 
+#	include <AudioToolbox/AudioToolbox.h>
+
 #	include "AudioInput.h"
 #	include "AudioOutput.h"
 
-#	include <AudioToolbox/AudioToolbox.h>
-#	include <Carbon/Carbon.h>
-
-#	include "Global.h"
+enum AUElement { OUTPUT = 0, INPUT = 1 };
```

Comment:
What is an `AUElement`?
Could we maybe use an enum class instead?

Reply:
`AUElement` is what this enum is called in Chromium's code. Is there any name you prefer?

Will turn them into enum class.

Reply:
I'm not entirely sure what an `AUElement` is. But from the options given inside I would probably call it
```suggestion
enum class Audio { OUTPUT = 0, INPUT = 1 };
```
' cause then you would use it as `Audio::INPUT` and `Audio::OUTPUT` respectively which (without knowing further details on where this is used) would read somewhat intuitive (imo).

Note however that if you have to convert it to and from integer values then using an enum class is not well suited :point_up:

Reply:
In some of my helper functions, I passed `AUElement` as a parameter. If I named it as `Audio`, the function declaration will look ambiguous like `xxxx(..., Audio direction, ...)` .

Personally, I prefer `AUDirection`, which means the direction parameter for Audio Units (AU). Is that okay?

Reply:
Ah okay yeah in that case your suggestion is indeed better. I think I would prefer simply calling it `AudioDirection` in order to not rely on implicit knowledge of abbreviations but given that AU is used in a lot of places in the code I think it's fine to use it here as well.